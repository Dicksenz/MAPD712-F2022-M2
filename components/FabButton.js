import React from "react";
import { View, StyleSheet, TouchableOpacity } from "react-native";
import Icon from "@expo/vector-icons/MaterialCommunityIcons";

const FabButton = () => {
  return (
    <TouchableOpacity onPress={() => {}}>
      <View style={styles.fabStyle}>
        <Icon name="plus" color="white" size={30} />
      </View>
    </TouchableOpacity>
  );
};

const styles = StyleSheet.create({
  fabStyle: {
    position: "absolute",
    justifyContent: "center",
    alignItems: "center",
    right: 12,
    bottom: 0,
    height: 70,
    width: 70,
    backgroundColor: "#347174",
    borderRadius: 100,
  },
});

export default FabButton;
