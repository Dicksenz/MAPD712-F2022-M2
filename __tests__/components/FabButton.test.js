import React from "react";
import renderer from "react-test-renderer";
import FabButton from "../../components/FabButton";

test("Check if FabButton renders correctly", () => {
  const tree = renderer.create(<FabButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
