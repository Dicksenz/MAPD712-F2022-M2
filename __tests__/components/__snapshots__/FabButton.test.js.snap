// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if FabButton renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#347174",
      "borderRadius": 100,
      "bottom": 70,
      "height": 70,
      "justifyContent": "center",
      "position": "absolute",
      "right": 12,
      "width": 70,
    }
  }
>
  <Text />
</View>
`;
